Cannot build kernel module into container with kernel 4.6.2

I'm trying to run sysdig with docker. I do (step by step).

```
docker pull sysdig/sysdig

docker run -i -t --rm --name sysdig --privileged \
  -v /var/run/docker.sock:/host/var/run/docker.sock \
  -v /dev:/host/dev \
  -v /proc:/host/proc:ro \
  -v /boot:/host/boot:ro \
  -v /lib/modules:/host/lib/modules:ro \
  -v /usr:/host/usr:ro \
  -v /tmp/sysdig:/tmp/sysdig \
  sysdig/sysdig
```

Once I enter into docker I give this error.

```
* Setting up /usr/src links from host
* Unloading sysdig-probe, if present
rmmod: ERROR: Module sysdig_probe is not currently loaded
* Running dkms autoinstall

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area....
make -j8 KERNELRELEASE=4.6.2-1-ARCH -C /lib/modules/4.6.2-1-ARCH/build M=/var/lib/dkms/sysdig/0.10.0/build....(bad exit status: 2)
Error! Bad return status for module build on kernel: 4.6.2-1-ARCH (x86_64)
Consult /var/lib/dkms/sysdig/0.10.0/build/make.log for more information.
* Trying to load a system sysdig-probe, if present
* Trying to load a dkms sysdig-probe, if present
* Trying to find precompiled sysdig-probe for 4.6.2-1-ARCH
Found kernel config at /proc/config.gz
* Trying to download precompiled module from https://s3.amazonaws.com/download.draios.com/stable/sysdig-probe-binaries/sysdig-probe-0.10.0-x86_64-4.6.2-1-ARCH-85c16288b73ee907ee84652179133196.ko
Download failed, consider compiling your own sysdig-probe and loading it or getting in touch with the sysdig community
```

Into make log can see the error.

```
DKMS make.log for sysdig-0.10.0 for kernel 4.6.2-1-ARCH (x86_64)
Sun Jun 12 10:22:07 UTC 2016
make: Entering directory '/host/lib/modules/4.6.2-1-ARCH/build'
  LD      /var/lib/dkms/sysdig/0.10.0/build/built-in.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/main.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/dynamic_params_table.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/flags_table.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/ppm_fillers.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/event_table.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/ppm_events.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/ppm_cputime.o
  CC [M]  /var/lib/dkms/sysdig/0.10.0/build/syscall_table.o
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/flags_table.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/flags_table.o] Error 127
make[1]: *** Waiting for unfinished jobs....
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/ppm_cputime.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/ppm_cputime.o] Error 127
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/dynamic_params_table.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/dynamic_params_table.o] Error 127
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/event_table.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/event_table.o] Error 127
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/syscall_table.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/syscall_table.o] Error 127
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/main.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/main.o] Error 127
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/ppm_events.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/ppm_events.o] Error 127
./tools/objtool/objtool: error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
scripts/Makefile.build:291: recipe for target '/var/lib/dkms/sysdig/0.10.0/build/ppm_fillers.o' failed
make[1]: *** [/var/lib/dkms/sysdig/0.10.0/build/ppm_fillers.o] Error 127
Makefile:1429: recipe for target '_module_/var/lib/dkms/sysdig/0.10.0/build' failed
make: *** [_module_/var/lib/dkms/sysdig/0.10.0/build] Error 2
make: Leaving directory '/host/lib/modules/4.6.2-1-ARCH/build'
```

The main problem is that into container `libelf` is missing. The workaround is install the missing library and re-run `/usr/bin/sysdig-probe-loader`

```
apt-get update
apt-get install libelf1
/usr/bin/sysdig-probe-loader
```

I think that docker image must include this missing library.

--- a/docker/dev/Dockerfile
+++ b/docker/dev/Dockerfile
@@ -20,7 +20,8 @@ RUN apt-get update \
 	curl \
 	ca-certificates \
 	gcc \
-	gcc-4.9 && rm -rf /var/lib/apt/lists/*
+	gcc-4.9 \
+	libelf1 && rm -rf /var/lib/apt/lists/*
 
 # Terribly terrible hacks: since our base Debian image ships with GCC 5.0 which breaks older kernels,
 # revert the default to gcc-4.9. Also, since some customers use some very old distributions whose kernel

--- a/docker/stable/Dockerfile
+++ b/docker/stable/Dockerfile
@@ -20,7 +20,8 @@ RUN apt-get update \
 	curl \
 	ca-certificates \
 	gcc \
-	gcc-4.9 && rm -rf /var/lib/apt/lists/*
+	gcc-4.9 \
+	libelf1 && rm -rf /var/lib/apt/lists/*
 
 # Terribly terrible hacks: since our base Debian image ships with GCC 5.0 which breaks older kernels,
 # revert the default to gcc-4.9. Also, since some customers use some very old distributions whose kernel

